### PR TITLE
selenium-requests package added

### PR DESCRIPTION
**Pull request by tsenapathi**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/selenium-requests/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python
+    - requests
```
Comment (bgruening) [inline recipes/selenium-requests/meta.yaml]: please remove all but pip and python

File: recipes/selenium-requests/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    - tldextract
+
+test:
+  imports:
+    - seleniumrequests
+
+about:
+  home: https://github.com/cryzed/Selenium-Requests
+  license: MIT
+  license_family: MIT
```
Comment (bgruening) [inline recipes/selenium-requests/meta.yaml]: add `license_file: LICENSE`

**Comment by tsenapathi** [comment]
Thanks @bgruening , Done the changes !!

**Comment by tsenapathi** [comment]
@bgruening should I remove `license_file: LICENSE `?

**Pull request merged**